### PR TITLE
Update Styleguide

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Update to latest Styleguide major version.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Invoices.js [hunk 1]
```diff
@@ -294,7 +294,7 @@ class Invoices extends Component {
           </div>
         )}
       </div>
-    )
+    );
```
Comment (leticiawanderley) [inline react/components/Invoices.js]: ⛏ remover

File: react/invoice.js [hunk 2]
```diff
@@ -304,14 +312,18 @@ class Invoice extends Component {
             <div className="tr modal-actions">
               <span className="mr4">
                 <Button
-                  size="large"
-                  secondary
+                  size="small"
```
[the remark this answers is not in the document]
Comment (kevinch) [reply react/invoice.js]: Sim. Tudo certo mesmo.

File: react/accounts.js [hunk 3]
```diff
@@ -566,13 +566,24 @@ class CheckingAccounts extends Component {
 
             {isFiltered ? (
               <div style={{ marginLeft: '-36px' }}>
-                <Button neutral icon type="button" onClick={this.handleReset}>
+                <Button
+                  size="small"
+                  variation="tertiary"
+                  neutral
```
Comment (leticiawanderley) [inline react/accounts.js]: Remover o neutral

File: react/accounts.js [hunk 4]
```diff
@@ -566,13 +566,24 @@ class CheckingAccounts extends Component {
...
                   <Deny color="currentColor" />
                 </Button>
               </div>
             ) : (
               <div style={{ marginLeft: '-36px' }}>
-                <Button neutral icon onClick={this.handleGetAccountsForm}>
+                <Button
+                  size="small"
+                  variation="tertiary"
+                  neutral
```
Comment (leticiawanderley) [inline react/accounts.js]: Idem

File: react/accounts.js [hunk 5]
```diff
@@ -929,7 +950,7 @@ class CheckingAccounts extends Component {
           </EmptyState>
         )}
       </div>
-    )
+    );
```
Comment (leticiawanderley) [inline react/accounts.js]: Remover

File: react/components/DataInlineUpdate.js [hunk 6]
```diff
@@ -126,20 +126,20 @@ class DataInlineUpdate extends React.Component {
...
                 </form>
-                <Button onClick={this.handleClose}>&times;</Button>
+                <Button size="small" variation="tertiary" onClick={this.handleClose}>&times;</Button>
               </div>
             )}
           </div>
         )}
       </div>
-    )
+    );
```
Comment (leticiawanderley) [inline react/components/DataInlineUpdate.js]: ⛏ remover

File: react/components/Holders.js [hunk 7]
```diff
@@ -306,20 +319,19 @@ class Holders extends Component {
...
             </span>
-            <Button size="large" primary onClick={this.handleDeleteHolder}>
+            <Button size="small" variation="primary" onClick={this.handleDeleteHolder}>
               {intl.formatMessage({ id: 'yes' })}
             </Button>
           </div>
         </Modal>
       </div>
-    )
+    );
```
Comment (leticiawanderley) [inline react/components/Holders.js]: Idem

File: react/components/UploadCsvToApi.js [hunk 8]
```diff
@@ -242,17 +242,17 @@ class UploadCsvToApi extends Component {
...
-                onClick={this.handleUpdateData}
-              >
+                onClick={this.handleUpdateData}>
                 {intl.formatMessage({ id: 'import.send.btn' })}
               </Button>
             </div>
           )}
       </div>
-    )
+    );
```
Comment (leticiawanderley) [inline react/components/UploadCsvToApi.js]: Idem

**Pull request merged**